### ISSUE TITLE
Only open source or free too?

### ISSUE THREAD

**Issue by joubertredrat** [n1trux/awesome-sysadmin#50]
Hi,

I'm looking that a lot of applications is open source, but have very nice free applications too, Axigen Free Mail Server as example, have enterprise solution, but have free version too and is very good.
https://www.axigen.com/mail-server/free/

The question is, why not to add free applications too and set this on list?
Example:
### Mailserver
* [Zimbra](https://www.zimbra.com/community/) - ```opensource,enterprise``` - Collaborative software suite, that includes an email server and web client.
* [Modoboa](http://modoboa.org/en/) - ```opensource``` - Modoboa is a mail hosting and management platform including a modern and simplified Web User Interface.
* [Axigen](https://www.axigen.com/mail-server/free/) - ```free,enterprise``` - The Axigen Free Mail Server is a great alternative to open source: it is a turnkey messaging solution with an amazing number of free email users.

**Comment by n1trux** [comment]
**tl;dr: nope, only open source.**

This list is specifically about open source software, which is the whole spirit of the "awesome-" movement. The point in this is that you can report bugs, fix them, add new features etc. without worrying about a restrictive license model.

There's a statement at the top of the list, quote:

>A curated list of amazingly awesome open source sysadmin resources

You can't fork freeware, which is not awesome in my eyes. I'd rather tighten the rules a little to set the focus to Free and Libre Open Source Software ([FLOSS](http://fsfe.org/freesoftware/basics/comparison.html)). To be honest, 95% of the software mentioned already adheres to this mentality.

Thanks a lot for your constructive input!

**Comment by joubertredrat** [comment]
Hi,
@n1trux , yes, then, why not a second list with software that isn't open source, but is free and can help SysAdmin too?

A list with this words as examples "These applications isn't awesomeness, but probably will help you if Awesome open source projects can't"
